raiden/utils/gas_reserve.py

```diff
@@ -92,7 +104,7 @@ def _get_required_gas_estimate_for_state(raiden) -> int:
 
 
 def has_enough_gas_reserve(
-    raiden,
+    raiden: 'RaidenService',
```

Comment:
Please don't add string types like that anymore. We will have to remove them as newer flake8 complains about them.